packages/landing/src/components/daterange.tsx

```diff
@@ -49,6 +50,20 @@ export class DateRange extends Component {
     this.scheduleUpdateConfig();
   };
 
+  componentDidMount(): void {
+    window.addEventListener('popstate', function () {
```

Comment:
This should not be done inside of the this class, you will get into lots of weird issues if you start b inding window events between multiple components. there is nothing stopping this component being added to the page multiple times.

The window URL / Config logic should be handling all of this.

If you ever bind events inside of a didMount, you need to make sure you unbind them too.

Reply:
I think this updateFromUrl already been added as event for the whole front-end. We might not need to add another listener for this. 
![image](https://user-images.githubusercontent.com/12163920/219499854-d06697a4-2402-4ce6-9e4f-8000c8670c89.png)